herbert_core/classes/MPIFramework/parallel_worker.m

```diff
@@ -25,14 +25,14 @@
 je = [];
 ok = false;
 err_mess = 'Failure in the initialization procedure';
+if ~exist('DO_LOGGING', 'var')
+    DO_LOGGING = false;
+end
+if nargin<3
```

Comment:
Why is this of a different form to `DO_LOGGING`?

Reply:
? why it is different. Its the same. If variable is not provided, it set to defaults. 
if `narin<3` is because Matlab is actually assign variables to names subsequently, so the call 
`parallel_worker(info,DO_DEBUGGING)` will be interpreted inside the routine as `parallel_worker(info,DO_LOGGING)`